pyrk/neutronics.py

```diff
@@ -84,15 +111,25 @@ def dpdt(self, t_idx, components, power, zetas):
         :type power: float.
         :param zetas: the current delayed neutron precursor populations, zeta_i
         :type zetas: np.ndarray.
+        :param rho_r: sum of the reactivity gain by all the reflectors
```

Comment:
This description does not match function signature (zeta_refs above, rho_r below)